src/main/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -267,7 +267,11 @@ public static double getGQLog10FromLikelihoods(int iOfChoosenGenotype, double[]
             String[] strings = GLString.split(",");
             double[] likelihoodsAsVector = new double[strings.length];
             for ( int i = 0; i < strings.length; i++ ) {
-                likelihoodsAsVector[i] = Double.parseDouble(strings[i]);
+                if ( strings[i].equals(VCFConstants.MISSING_VALUE_v4) ) {
+                  return null; // one or more entries are missing
```

Comment:
hmmm. so 1,.,2 will return a null value? I guess with primitive doubles we have, to...though we could use NaNs for that. maybe when we support 4.3....

Reply:
I think a mixed case like `1,.,2` should probably throw an exception.